perception/object_recognition/detection/roi_cluster_fusion/CMakeLists.txt

```diff
@@ -2,27 +2,30 @@ cmake_minimum_required(VERSION 3.5)
 project(roi_cluster_fusion)
 
 if(NOT CMAKE_CXX_STANDARD)
-	set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 14)
 endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
 ament_auto_find_build_dependencies()
```

Comment:
Could you move `ament_auto_find_build_dependencies()` after `find_package( PCL REQUIRED )`? This causes cmake warning as below.
`CMake Warning (dev) at CMakeLists.txt:17 (find_package):
  Policy CMP0074 is not set: find_package uses <PackageName>_ROOT variables.
  Run "cmake --help-policy CMP0074" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.
`

Reply:
fixed at 
5c6743e